### PR TITLE
[exporter/prometheusremotewrite] change feature gate to beta

### PR DESCRIPTION
**Pull request by perebaj**
#### Description

Change `exporter.prometheusremotewriteexporter.deprecateCreatedMetric` feature gate from Alpha to Beta

#### Link to tracking issue
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35003

### CODE REVIEW

File: .chloggen/exporter_prometheusremotewriteexporter_deprecate-created-metric-alpha2beta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Change `exporter.prometheusremotewriteexporter.deprecateCreatedMetric` feature gate from Alpha to Beta version.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [35003]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```
Comment (dashpole) [inline .chloggen/exporter_prometheusremotewriteexporter_deprecate-created-metric-alpha2beta.yaml:18]:
```suggestion
subtext: The `export_created_metric` configuration parameter is now ignored by default.
```

**Review by dashpole — APPROVED** [review]

**Comment by ArthurSens** [comment]
@perebaj or someone with permissions, could we remove the statement `Fixes #...` from the PR description? We want to close the issue only after the whole deprecation is done :)

**Comment by perebaj** [comment]
> @perebaj or someone with permissions, could we remove the statement `Fixes #...` from the PR description? We want to close the issue only after the whole deprecation is done :)

Make sense, removed

**Review by ArthurSens — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Comment by perebaj** [comment]
Guys, I need to do some follow up task here? 

After this, when we will be able to promote this gate to the next stage?

**Comment by dashpole** [comment]
I think that is it for now.  We should wait a few releases to see if there is any feedback or issues, now that it is enabled by default.  We can remove after ~3 more releases.

**Pull request merged**